server/src/main/java/org/candlepin/pki/SubjectKeyIdentifierWriter.java

```diff
@@ -36,6 +40,14 @@
      * @throws IOException thrown if error reading cert
      * @throws NoSuchAlgorithmException thrown if JcaX509ExtensionUtils can't be created
      */
-    byte[] getSubjectKeyIdentifier(KeyPair clientKeyPair, Set<X509ExtensionWrapper> extensions)
-        throws IOException, NoSuchAlgorithmException;
+    default byte[] getSubjectKeyIdentifier(KeyPair clientKeyPair, Set<X509ExtensionWrapper> extensions)
+            throws IOException, NoSuchAlgorithmException {
+        return new JcaX509ExtensionUtils().createSubjectKeyIdentifier(clientKeyPair.getPublic()).getEncoded();
+    }
+
+    default byte[] toOctetString(byte[] bytes_to_encode) throws IOException {
```

Comment:
Please use camel case for your parameters instead of snake care.  E.g. "byteToEncode" instead of "bytes_to_encode".  Snake case is the idiomatic pattern in Python, but Java favors camel case.